data-loader/core/src/main/java/com/scalar/db/dataloader/core/DataLoaderError.java

```diff
@@ -0,0 +1,292 @@
+package com.scalar.db.dataloader.core;
+
+import com.scalar.db.common.error.Category;
+import com.scalar.db.common.error.ScalarDbError;
+
+public enum DataLoaderError implements ScalarDbError {
```

Comment:
@josh-wong IIRC, you haven’t checked the error messages in data-loader yet. If that’s correct, could you please take a look? 🙇